### ISSUE TITLE
BarChart animation

### ISSUE THREAD

**Issue by FranciscoDaunas** [danielgindi/Charts#2616]
Hello, I'm working on a power graph using the Bar Chart. I have fixed the margins and I keep changing the values of the bars. I was wondering if there is any animation to move the bar from it's current position to the next value?

**Comment by liuxuan30** [comment]
no, but you are welcome to contribute one. Look at AnimatorJob to see any inspiration
